### ISSUE TITLE
Cannot execute PhantomJS from browser using PHP exec. CentOS 6.8

### ISSUE THREAD

**Issue by lincolnaleixo** [ariya/phantomjs#14406]
1. Which version of PhantomJS are you using?
   2.1.1
2. What steps will reproduce the problem?
   1. Created a PHP file test.php

test.php:

```
echo "Running PhantomJS version: ";
echo exec('/usr/local/bin/phantomjs /path_to_dir/test.js 2>&1');
```

test.js:

```
console.log('Hello, world!');
phantom.exit();
```

  2.Execute from terminal, everything works fine. Executing from browser I get this:

> Running PhantomJS version: and file a bug report.
1. Which operating system are you using?

CentOS release 6.8 (Final)
1. Did you use binary PhantomJS or did you compile it from source?

Binary
1. Please provide any additional information below.

I installed phantomjs using the following commands:

> wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2
> bunzip2 phantomjs-2.1.1-linux-x86_64.tar.bz2
> tar xf phantomjs-2.1.1-linux-x86_64.tar
> mv phantomjs-2.1.1-linux-x86_64/phantomjs /usr/local/bin

Everything works perfectly from terminal, the problem only occurs when executing from PHP using the browser (also works when using PHP from terminal)

**Comment by linkinkin** [comment]
Update 1:

Somethings I get this response:

`1 0x1b67967 phantomjs() [0x1b67967] 2 0x1baaaf5 phantomjs() [0x1baaaf5] 3 0x198c463 phantomjs() [0x198c463] 4 0x1ab3982 phantomjs() [0x1ab3982] 5 0x79abb9 phantomjs() [0x79abb9] 6 0x76b6f9 phantomjs() [0x76b6f9] 7 0x750a5d phantomjs() [0x750a5d] 8 0x41cd89 phantomjs() [0x41cd89] 9 0x41f5c4 phantomjs() [0x41f5c4] 10 0x415b86 phantomjs() [0x415b86] 11 0x7f0160e4cd1d /lib64/libc.so.6(__libc_start_main+0xfd) [0x7f0160e4cd1d] 12 0x4176a1 phantomjs() [0x4176a1] PhantomJS has crashed. Please read the bug reporting guide at and file a bug report.`

**Comment by zerosimms** [comment]
I had the same issue with Bluehost, managed to "fix" it by changing my PHP version to: PHP 5.4 (FastCGI) from PHP 5.4. Now I can execute my script through PHP fine. Hope this helps you.
